Neos.Neos/Resources/Private/Styles/FontAwesome/brands.scss

```diff
@@ -13,8 +13,7 @@
   src: url('#{$fa-font-path}/fa-brands-400.eot?#iefix') format('embedded-opentype'),
```

Comment:
At some point we can throw most of them out as we only need woff2 and maybe woff

Reply:
Yeah true... to be honest just removed it because the type of svg they used lead to issues 🤣